src/game/client/ingameui.cpp

```diff
@@ -13,35 +13,1131 @@
+                                            int seconds = countdown_seconds % 60;
+
+                                            Utf8String text;
+                                            text.Format("GUI:%s", name.Str());
+                                            Utf16String title;
+                                            Utf16String time;
+                                            title.Format(U_CHAR("%ls: "), g_theGameText->Fetch(text.Str()).Str());
+                                            time.Format(U_CHAR("%d:%2.2d"), minutes, seconds);
+                                            info->Set_Text(title, time);
+                                        }
+
+                                        if (!is_ready || m_superweaponFlashDuration == 0.0f) {
+                                            info->Draw_Name(superweapon_x, superweapon_y, 0, border_color);
+                                            info->Draw_Time(superweapon_x, superweapon_y, 0, border_color);
+                                        } else {
+                                            if (g_theGameLogic->Get_Frame()
+                                                >= (unsigned int)m_superweaponFlashDuration + m_superweaponLastFlashFrame) {
+                                                m_superweaponHidden = !m_superweaponHidden;
+                                                m_superweaponLastFlashFrame = g_theGameLogic->Get_Frame();
+                                            }
+
+                                            info->Draw_Name(superweapon_x,
+                                                superweapon_y,
+                                                m_superweaponHidden ? 0 : m_superweaponFlashColor,
+                                                border_color);
+                                            info->Draw_Time(superweapon_x,
+                                                superweapon_y,
+                                                m_superweaponHidden ? 0 : m_superweaponFlashColor,
+                                                border_color);
+                                        }
+
+                                        superweapon_y += info->Get_Height();
+                                    }
+
+                                    if (info->Get_Special_Power_Template()->Get_Shared_Synced_Timer()) {
+                                        break;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    if (g_theGameLogic->Get_Frame() != 0 && m_showNamedTimers) {
+        bool is_right_aligned = m_namedTimerPosition.x >= 0.5f;
+        int timer_x = g_theDisplay->Get_Width() * m_namedTimerPosition.x;
+        int timer_y = g_theDisplay->Get_Height() * m_namedTimerPosition.y;
+        int border_color = Make_Color(0, 0, 0, 255);

```

Comment:
This can be constexpr because `Make_Color` is constexpr and all inputs are constants.